app/src/main/java/com/github/se/signify/ui/screens/profile/SettingsScreen.kt

```diff
@@ -187,11 +171,26 @@ fun SettingsScreen(
                         cursorColor = MaterialTheme.colorScheme.onBackground))
           }
     }
-    Spacer(modifier = Modifier.height(32.dp))
+    Spacer(modifier = Modifier.height(64.dp))
+
+    Row(
+        modifier = Modifier.fillMaxWidth().padding(16.dp).clickable { onThemeChange(!isDarkTheme) },
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween) {
+          val modeText =
+              if (isDarkTheme) stringResource(R.string.dark_mode_text)
+              else stringResource(R.string.light_mode_text)
+          Text(
+              text = modeText,
+              style = MaterialTheme.typography.bodyLarge,
+              color = MaterialTheme.colorScheme.onBackground)
+          Switch(checked = isDarkTheme, onCheckedChange = { onThemeChange(it) })
+        }
+    // Switch between English and French
+    LanguageSwitch(isFrench, onLanguageChange)
+
+    Spacer(modifier = Modifier.height(64.dp))
```

Comment:
If I remember well , we discussed to move those switch under the username, i prefer it like this!